src/YoutubeAtomPlayer.tsx

```diff
@@ -75,7 +75,34 @@ const createOnStateChangeListener = (
      */
     const player = event.target;
 
+    /**
+     * the ID of the iframe
+     */
+    const atomId = `youtube-video-${videoId}`;
+
     if (event.data === YT.PlayerState.PLAYING) {
+        const youtubeIframes = document.querySelectorAll(
+            '[data-atom-type="youtube"]',
+        );
+
+        /**
+         * send a 'stopVideo' postMessage to all other YoutubeAtoms
+         */
+        Array.from(youtubeIframes).forEach((iframe) => {
+            if (iframe instanceof HTMLIFrameElement) {
+                if (iframe?.id !== atomId) {
```

Comment:
```suggestion
                if (iframe && iframe.id !== atomId) {
```

Ginormous nitpick so feel free to ignore this if you are even slightly not feeling it but maybe this reads nicer?

Reply:
It might be that this isn't needed in either form - see the check on the line above

Reply:
@joecowton1 do you mean we can just get away with `iframe.id` instead of `iframe && iframe.id` or `iframe?.id`?

Reply:
yeah I _think_ so

Reply:
yea makes sense to me 👍